packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/web-services/socket-session-endpoint.ts

```diff
@@ -0,0 +1,203 @@
+import { Socket as SocketIoSocket } from "socket.io";
+
+import { Logger, Checks } from "@hyperledger/cactus-common";
+import { LogLevelDesc, LoggerProvider } from "@hyperledger/cactus-common";
+import {
+  RunTransactionRequestV1,
+  RunTransactionResponse,
+  IrohaBlockProgress,
+  IrohaCommand,
+  IrohaBaseConfig,
+} from "../generated/openapi/typescript-axios";
+import {
+  IrohaSocketSession,
+  IrohaQuery,
+} from "../generated/openapi/typescript-axios";
+
+import { Transaction } from "../transaction";
+export interface IWatchBlocksV1EndpointOptions {
+  logLevel?: LogLevelDesc;
+  socket: SocketIoSocket;
+}
+
+export class SocketSessionEndpoint {
```

Comment:
@StepniowskiP Any chance that you could make the name of the class and the constructor arg interface type consistent?
`SocketSessionEndpoint` is a bit too generic in my opinion because literally every endpoint that uses SocketIO could be accurately be described by the same name.

Reply:
Sure, renamed to iroha-socketio-endpoint